plugins/parquet/parquetembed.cpp

```diff
@@ -844,9 +844,6 @@ void ParquetWriter::beginSet(const RtlFieldInfo *field)
     arrow::ArrayBuilder *childBuilder;
     arrow::FieldPath match = getNestedFieldBuilder(field, childBuilder);
     fieldBuilderStack.push_back(std::make_shared<ArrayBuilderTracker>(field, childBuilder, CPNTSet, std::move(match)));
-
-    arrow::ListBuilder *listBuilder = static_cast<arrow::ListBuilder *>(childBuilder);
```

Comment:
This piece of code pads the list builder and sets it up for adding list elements to it. I believe what was meant to go here was a change from a ListBuilder to LargeListBuilder.

Reply:
Make sure that you are testing writing Parquet files with an ECL SET as this change would cause an error.